### PR TITLE
Add instructions for debug toasts

### PR DESCRIPTION
**Pull request by anton-kapralov**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -388,6 +388,28 @@ successful integration.
 | `liftoff-rewarded-video-test`     | Rewarded Interstitial | VAST video             |
 | `liftoff-native-test`             | Native                | Native                 |
 
+#### Debugging failures
```
Comment (zdyn) [inline README.md:391]: FYI, we have a "Troubleshooting" section that would be a better section for this.

File: README.md [hunk 2]
```diff
@@ -388,6 +388,28 @@ successful integration.
...
+
+It is possible to turn on the debug mode and observe ads failures as toasts. In
+order to see those toasts with error messages you need the following:
```
Comment (zdyn) [inline README.md:394]: Slightly less awkward phrasing:
```suggestion
You can observe the reason for an ad load failure as a toast message.
To enable toast messages, turn on debug mode:
```

File: README.md [hunk 3]
```diff
@@ -388,6 +388,28 @@ successful integration.
...
+  withMediatedNetworkConfiguration(
+    LiftoffAdapterConfiguration::class.java.name,
+    mapOf(
+      "apiKey" to LIFTOFF_API_KEY, 
+      "debug" to "true" // <--
+    )
+  )
+}
+
+// Or set the field directly if you use self-mediated setup.
```
Comment (zdyn) [inline README.md:409]:
```suggestion
// Or set the field directly if you use a self-mediated setup.
```

**Review by zdyn — CHANGES_REQUESTED** [review]
This functionality isn't available yet, right? IIUC, we need to release a patch update and mention that this feature requires v1.3.1.

**Comment by anton-kapralov** [comment]
Thanks!

Yes, added "Requires v1.3.1+." at the beginning of the section.

**Review by zdyn — APPROVED** [review]

**Comment by anton-kapralov** [comment]
Thanks!

**Pull request merged**